nepali/about_nepal.md

```diff
@@ -0,0 +1 @@
+नेपाल एक सुनदर देश हो । यहाँका पाकृितक समपदा र पुरातातिवक समपदाहरूको बखान अनतराषिटय समुदायमा पिन ठुलो छ । थप जानकारीको लािग [यहाँ िथचनुहोस्] । (http://en.wikipedia.org/wiki/Nepal)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.